src/domains/chart/chart-types.ts

```diff
@@ -70,7 +70,7 @@ export interface ChartMetadata {
   dimensions: {
     [key: string]: Dimension
   }
-  chartLabels?: {
```

Comment:
We don't always have that in `ChartMetadata` payload.

You can instead add question mark back, and in `render-charts-and-menu` use `!`, for example: `chartEnriched.chartLabels!.k8s_cluster_id` by which you signalise that this property is not `undefined`, neither `null`.